azurelinuxagent/agent.py

```diff
@@ -317,6 +317,17 @@ def setup_firewall(endpoint):
 
         try:
             firewall_manager = FirewallManager.create(endpoint)
+            if isinstance(firewall_manager, IpTables):
+                try:
+                    #
+                    # We execute "iptables -C -m conntrack" to force loading of the conntrack module with the intention of avoiding the
+                    # issue described in IpTables.check().
+                    #
+                    logger.info("Attempting to load the conntrack module (iptables -C should not find any matching rules, so that error can be ignored)...")
+                    logger.info("{0}", firewall_manager.load_conntrack())
```

Comment:
will this work running -C command on drop rule will load conntrack module? I guess not because it didn't happen when agent used -C command in the main loop. If it had happened, env thread would have been successful in detecting drop rule

Reply:
the idea here is to load the module before creating (-A) the rule . see the sample log I posted in our internal chat

Reply:
ok, -A will not load the module and only -C does that?